ontology/yaml/resources/HVAC/entity_types/DWST.yaml

```diff
@@ -26,6 +26,40 @@ DWST_CWDT_SWTC_CPC:
   - potable_water_medium_temperature_sensor
   - potable_water_low_temperature_sensor
 
+DWST_HWDT_CPC_SWTC2X_HWFM_HSWPM:
+  guid: "cd74f84e-969a-422c-8c84-d40ca6281352"
+  description: "Non-standard domestic water system with two heating water heat exhangers that are manually opened/closed, 
+  one circulation pump, and heating water system flow and btu monitoring"
+  is_canonical: false
```

Comment:
You need to place the non_canonical types in the section below

Reply:
I added a section for non_canonical types and moved all non_cononical to this section